code-security/admin_guide/scan-monitor/sbom-generation.adoc

```diff
@@ -1,83 +1,112 @@
+
+Prisma Cloud scans your integrated repositories for open source packages through the package manager files found in the repository and IaC resources to generate an SBOM report. You can choose to generate the SBOM report either through a CI/CD pipeline or the console. In addition, you can also choose the SBOM generation format to be either a CSV or a CycloneDX. 
 
-On Prisma Cloud you can export open source packages found in package manager files or container images, as well as IaC resources and findings to generate an SBOM report. An SBOM report can either be an artifact from a CI/CD pipeline or from the Prisma Cloud platform for onboarded repositories.
 
 toc::[]
 
-[.task]
+=== Types of SBOM Reports
 
-=== Generate an SBOM on the Prisma Cloud Console
+Prisma Cloud currently supports the export of SBOM reports in two standardized formats, CSV and CyclondeDX.
 
-You can access the SBOM Generation wizard from either the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/supply-chain-security[Supply Chain] page or the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/development-pipelines[Development Pipelines] page.
+* CSV
++
+CSV is a comma-separated format that displays a line per vulnerability or misconfiguration. This format includes a line per resource or a package without vulnerabilities with categories across open source packages, container images, IaC components, and cloud infrastructure components.
 
+* CycloneDX output follows the scheme outlined for XML version 1.4.
++
+CycloneDX format displays vulnerabilities or misconfiguration as a scheme outlined for XML version 1.4.
 
-image::sbom-1.png[width=600]
+==== Generate SBOM as a CSV
 
-image::sbom-2.png[width=600]
+Generating an SBOM in a CSV format from the CI/CD pipeline will generate three files:
 
-[.procedure]
+* YYYYMMDD-HHMMSS_iac.csv
+* YYYYMMDD-HHMMSS_container_images.csv
+* YYYYMMDD-HHMMSS_oss_packages
 
+Here YYYYMMDD-HHMMSS is the timestamp of the CSV generation. 
 
-. Select the repository you would like to include in the report.
+In this example, 20220801-221830_iac.csv. This file will display the following data for each IaC scan, including resources without misconfigurations:
 
-. Select the output format (CSV or CycloneDX).
+* Resource name
+* Path
+* Git Organization
+* Git Repository
+* Misconfigurations
+* Severity
 
-. Select which resources to include in the SBOM:
+==== Generate SBOM as a CycloneDX
 
-* *Open-source packages* - findings from scans of open source dependencies
-* *Infrastructure as Code* - findings from IaC scans
-* *Images* - packages found in images built or referenced in the repository
+Generating an SBOM in a CycloneDX format will generate a single file with sections like:
```

Comment:
These are the sections, so not "like"

`Generating an SBOM in a CycloneDX format will generate a single file with the following sections:`

Reply:
updated.